v03_pipeline/lib/misc/validation.py

```diff
@@ -104,8 +154,17 @@ def _validate_field(
 
 def validate_imputed_sex_ploidy(
     mt: hl.MatrixTable,
-    sex_check_ht: hl.Table,
+    dataset_type: DatasetType,
+    skip_check_sex_and_relatedness: bool,
+    sex_check_ht: hl.Table | None = None,  # nb: sex_check_ht will be undefined if
+    **_: Any,
 ) -> None:
+    if not (
```

Comment:
instead of writing the same conditional check here and in `get_validation_dependencies` you could just check here if `sex_check_ht` is None